src/index.json

```diff
@@ -2120,11 +2120,10 @@
         ],
         "webapp": [
             {
-                "downloadUrl": "https://github.com/patricklee2/extensions-whl/raw/master/webapp-0.2.17-py2.py3-none-any.whl",
-                "filename": "webapp-0.2.17-py2.py3-none-any.whl",
+                "downloadUrl": "https://github.com/panchagnula/azure-cli-extensions/raw/sisirap-extensions-whl/dist/webapp-0.2.18-py2.py3-none-any.whl",
+                "filename": "webapp-0.2.18-py2.py3-none-any.whl",
                 "metadata": {
                     "azext.isPreview": true,
-                    "azext.minCliCoreVersion": "2.0.46",
```

Comment:
Have you talked to @limingu regarding adding maxCliCoreVersion 2.0.66?

Reply:
this is done.